### PR TITLE
Wrap requester values so they truncate

### PR DESCRIPTION
**Pull request by rhamilto**
After:
screen shot 2019-01-08 at 9 45 32 am

### CODE REVIEW

File: frontend/public/components/namespace.jsx [hunk 1]
```diff
@@ -92,7 +92,7 @@ const ProjectRow = ({obj: project}) => {
     <div className="col-md-3 col-sm-3 col-xs-4">
       <StatusIcon status={project.status.phase} />
     </div>
-    <div className="col-md-3 col-sm-3 hidden-xs">
+    <div className="col-md-3 col-sm-3 hidden-xs co-break-word">
```
Comment (spadgett) [inline frontend/public/components/namespace.jsx]: There's a fix for this in #865 (although break word might be better than truncate)

**Comment by rhamilto** [comment]
Rebased

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Pull request merged**